docs/src/content/docs/fr/resources/community-content.mdx

```diff
@@ -74,7 +74,11 @@ Explorez le contenu communautaire produit par les utilisateurs de Starlight :
 		title="Publier une documentation avec Astro Starlight"
 		description="Démarrer une documentation avec Starlight"
 	/>
-
+	<LinkCard
+		href="https://events-3bg.pages.dev/jotter/starlight/guide/"
+		title="Utiliser les transitions de vue (view transitions)"
```

Comment:
Used the same convention as the Astro Docs (https://docs.astro.build/fr/guides/view-transitions/)